crates/bcr-ebill-wasm/src/context.rs

```diff
@@ -44,12 +44,14 @@ impl Context {
         ));
         let bitcoin_client = Arc::new(BitcoinClient::new());
 
-        let nostr_client = create_nostr_client(&cfg, db.identity_store.clone()).await?;
+        let nostr_clients =
```

Comment:
Is there a way to update these at runtime? :thinking: 

E.g. if a new company is created / we're added as a signatory?

Or would we need to do a refresh in that case?

Reply:
Nope, we need a reload for that atm. We can probably make this mutable but it is a bit more complex then.

Reply:
If it's in the context, it's tricky, since that's now a static reference. I guess it could be a RefCell inside leveraging internal mutability. :thinking: 
But for now a refresh is probably fine as well. Just something to think about for the future

Reply:
The tricky part for me is, what to do with the join handle. The subscribe has already returned the joinset and main is awaiting it.